packages/auth-client/src/index.ts

```diff
@@ -338,7 +338,7 @@ export class AuthClient {
               );
             }
           } catch (err) {
-            this._handleFailure(err.message, options?.onError);
+            this._handleFailure((err as Error).message, options?.onError);
```

Comment:
would this ever not be an `Error` type? would optional chaining be useful here too?

```typescript
this._handleFailure((err as Error)?.message, options?.onError);
```

Reply:
It will always be an Error, but Typescript insists that it has to be treated as `undefined`